pkgs/development/python-modules/durus/default.nix

```diff
@@ -14,6 +15,13 @@ buildPythonPackage rec {
     hash = "sha256:1gzxg43zawwgqjrfixvcrilwpikb1ix9b7710rsl5ffk7q50yi3c";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/nascheme/durus/commit/501a802d3763a6091b97d2f7737e59bf133e88ba.patch";
```

Comment:
This patch is [available](https://github.com/nascheme/durus/commit/ec9afa891aed851ed532da8624f69938e0988f43) in 4.3. Upstream just forgot to tag it.